### ISSUE TITLE
changelog: v86

### ISSUE THREAD

**Issue by Thorin-Oakenpants** [arkenfox/user.js#1127]
* date: 28-Feb-2021
* foreword: These are all the changes since the last changelog ([v85](https://github.com/arkenfox/user.js/issues/1104)).

[FF86 release notes](https://www.mozilla.org/firefox/86.0/releasenotes/)
[FF86 for developers](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/86)
[FF86 security advisories](https://www.mozilla.org/en-US/security/advisories/mfsa2021-07/)

---

### CHANGELOG: [[all changes](https://github.com/arkenfox/user.js/compare/85.0...86.0)]
* all user.js **updates for Firefox v86** are detailed in the [ToDo: diffs FF85-FF86](https://github.com/arkenfox/user.js/issues/1117) issue
  * includes links to the commits made for each pref, links to bugzilla tickets, our discussions etc.
* for all the rest see the full list of pref changes below

---

### ALL PREF CHANGES

* new in user.js v86
   - visited links was removed in [v84](https://github.com/arkenfox/user.js/issues/1088) but has been re-added if you want to harden: see https://github.com/arkenfox/user.js/commit/612cfbf3134d8685a1448d9dc2500dd8b91feb6e
```js
   // user_pref("browser.quitShortcut.disabled", true); // disable Ctrl-Q quit shortcut [LINUX] [MAC] [FF87+]
   // user_pref("layout.css.visited_links_enabled", false);
```

* changed in user.js v86
```js
user_pref("dom.popup_allowed_events", "click dblclick mousedown pointerdown"); // prev "click dblclick"
```

* made inactive in user.js v86
   - for widevine: see #1107 [it is already covered by `1830`]
   - for ssl session ids: see #1110
   - for webgl: see https://github.com/arkenfox/user.js/commit/4596d721e6479aa2341554dc9e21cd999cd704a8
```js
   // user_pref("media.gmp-widevinecdm.enabled", false);
   // user_pref("security.ssl.disable_session_identifiers", true); // [HIDDEN PREF]
   // user_pref("webgl.min_capability_mode", true);
```

* removed from user.js v86 - see #1107 [it is already hidden by `1830`]
```js
user_pref("media.gmp-widevinecdm.visible", false);
```

* moved to `9999: DEPRECATED / REMOVED`:
```js
user_pref("browser.download.hide_plugins_without_extensions", false);
user_pref("security.ssl.errorReporting.automatic", false);
user_pref("security.ssl.errorReporting.enabled", false);
user_pref("security.ssl.errorReporting.url", "");
```

---

### STATS

```
 STATS v86
 =========
 up to and including section 4500, minus the parrots

    total: 320
 inactive: 106
           ---
   active: 214
  default:  14 (at least)
           ---
  flipped: 200
      n/a:   2 (of the three prefs in 0204, only one will apply)
           ---
   effect: 198 (at most)
```

---

Any and all help, suggestions, recommendations, links, tips and tricks, questions, thank you's or what have you are welcome - signup/login and start typing - thanks

**Comment by Thorin-Oakenpants** [comment]
out of interest

### TL;DR

```
               TOT ACTIVE INACTIVE EFFECT
v60  MAY-2018  425   328      97    316
v68  JUL-2019  339   245      94    227
v78  JUN-2020  330   229      99    214
v86  FEB-2021  320   214     106    198
```

Trending in the right direction
- RFP slowly making more items redundant and shoved to the RFP ALTS
- defaults changing to what we use
- old code stripped out and permanent privacy enhancing changes are made
- visionaries like Pants keeping shit simple and clean 👍 

Of course, if you'd like to have 425 prefs again ... I'm sure some crappy repos and forks will oblige. Class, discuss!

---

### FF60 May 2018

```
 STATS v60
 =========
    total: 425
 inactive:  97
           ---
   active: 328
  default:  12 (at least)
           ---
  flipped: 316
      n/a:   0
           ---
   effect: 316 (at most)
```

### JULY 2019 : FF68

There were a lot of removals of what I considered to be "dead wood" or redundant prefs around v67 (and v68), not to mention plenty of changes based on people's overrides: and someone claimed [I had officially lost it](https://github.com/arkenfox/user.js/issues/732#issue-449332096) .. it's so hard being the visionary, *sigh*

```
 STATS v68
 =========
    total: 339
 inactive:  94
           ---
   active: 245
  default:  16 (at least)
           ---
  flipped: 229
      n/a:   2 (of the three prefs in 0211, only one will apply)
           ---
   effect: 227 (at most)
```

### JUNE 2020 : FF78

```
 STATS v78
 =========
    total: 330
 inactive:  99
           ---
   active: 231
  default:  15 (at least)
           ---
  flipped: 216
      n/a:   2 (of the three prefs in 0204, only one will apply)
           ---
   effect: 214 (at most)
```

**Comment by Marc05** [comment]
I'd just like to say, since it seems comments are welcome, that my list of user-overrides.js grows shorter and shorter. Thank you for continuing this effort!

**Comment by atomGit** [comment]
ditto - my list is still lengthy, but i'm not overriding quite as many user.js prefs any longer and i haven't had to troubleshoot web breakage due to prefs in quite a while

**Comment by Thorin-Oakenpants** [comment]
my overrides is pretty consistent: it doesn't seem to shrink or grow. At quick glance about `6` are not in the user.js, about `14` items from those in 4500 and lower, and the rest are all 5000s or something: I should do a bit of an analysis, but I do like a clean minimalist UI and no fucking distractions :)

> since it seems comments are welcome

comments are **_always_** welcome, especially when they are praiseworthy  😀  flattery will get you everywhere

**Comment by Thorin-Oakenpants** [comment]
FYi: https://bugzilla.mozilla.org/show_bug.cgi?id=1680237
- I like this, now your one-off PB mode windows can behave differently

**Comment by Thorin-Oakenpants** [comment]
interesting read: https://blog.mozilla.org/attack-and-defense/2021/03/10/insights-into-https-only-mode/
